builder/builder.go

```diff
@@ -129,11 +150,21 @@ func (b *Builder) Start() error {
 				} else if payloadAttributes.Slot == currentSlot {
 					// Subsequent sse events should only be canonical!
 					if !b.ignoreLatePayloadAttributes {
-						b.OnPayloadAttribute(&payloadAttributes)
+						err := b.OnPayloadAttribute(&payloadAttributes)
```

Comment:
The function already handles logging, I don't think this is necessary.
Otherwise you should also remove the logging from the hook, and wrap the errors instead.

Reply:
good catch, discussed offline - wrapped the errors and removed logging from the hook, please feel free to un-resolve if i missed anything